cordova-lib/src/platforms/platforms.js

```diff
@@ -102,5 +102,10 @@ function getPlatformProject(platform, platformRootDir) {
 }
 
 module.exports = platforms;
-module.exports.getPlatformProject = getPlatformProject;
-module.exports.PlatformProjectAdapter = PlatformProjectAdapter;
+
```

Comment:
Oops, I added those but missed them in the platform list.
getPlatformProject() is used in several places throughout the code and PlatformProjectAdapter is used by a downstream project, so we need them.
Need to find a way to "hide" them for the platform listing.